uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/leave-request-popup-controller_test.js

```diff
@@ -280,7 +306,7 @@
 
         describe('when to date is selected', function () {
           beforeEach(function () {
-            $ctrl.onDateChange(new Date(), 'to');
+            setDates2016(true, true);
```

Comment:
Its better to create 2 functions
`setFromDatesTo(<datevariable>)`
`setToDatesTo(<datevariable>)`

It will support all dates, not only 2016. Also will eliminate passing params which are not clear to layman.